<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionMessageDialog</class>
 <widget class="QDialog" name="GestionMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Messages</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Type de message :</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="promotionRadio">
            <property name="text">
             <string>Promotion</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="pointAppelRadio">
            <property name="text">
             <string>Point appel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ouvertureRadio">
            <property name="text">
             <string>(Pré) Ouverture</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QRadioButton" name="programmesRadio">
            <property name="text">
             <string>Programmés</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="jingleRadio">
            <property name="text">
             <string>Jingle</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="fermetureRadio">
            <property name="text">
             <string>(Pré) Fermeture</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="2">
        <widget class="QPushButton" name="supAncienButton">
         <property name="text">
          <string>Supprimer les messages sélectionnés</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="supActuelButton">
         <property name="text">
          <string>Supprimer les messages sélectionnés</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="actuelLabel">
         <property name="text">
          <string>Actuelle</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="leftToRightButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rightToLeftButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="ancienLabel">
         <property name="text">
          <string>Ancien</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="2">
        <widget class="QListView" name="actuelList"/>
       </item>
       <item row="2" column="2" rowspan="2">
        <widget class="QListView" name="ancienList"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Fermer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>GestionMessageDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
